docs/release.sh

```diff
@@ -158,4 +163,4 @@ sed -i "s/^site_url:.*/site_url: \/$MAJOR_MINOR\//" mkdocs.yml
 echo "Building the /$MAJOR_MINOR/ documentation"
 build_current_documentation
 upload_current_documentation "/$MAJOR_MINOR/"
-invalidate_cache "/$MAJOR_MINOR"
+[ "$NOCACHE" ] || invalidate_cache "/$MAJOR_MINOR"
```

Comment:
it's funny bc i did that here and forgot to fix the other, next time im shooting for 100% awesomeness